engine/src/engine.ts

```diff
@@ -205,12 +214,22 @@ export default class Engine {
 
   /** Fix old options passed. To remove when legacy data is no more in database */
   sanitizeOptions() {
+    if (this.options.factionVariant == null) {
```

Comment:
```
if (!this.options.factionVariant) {
```

Someone might accidentally convert `==` to `===` either manually or with a linting tool, and its `===` value when unset is `undefined`, not `null`.